ush/config.sh_rrfs_a_enkf_n3

```diff
@@ -0,0 +1,178 @@
+MACHINE="wcoss2"
+version="v0.3.7"
+ACCOUNT="RRFS_DEV"
+#RESERVATION="rrfsdet"
+EXPT_BASEDIR="/lfs/h2/emc/lam/noscrub/emc.lam/rrfs/${version}"
+EXPT_SUBDIR="rrfs_na_ens"
+
+PREDEF_GRID_NAME="RRFS_NA_3km"
+. set_rrfs_config_general.sh
+
+DO_ENSEMBLE="TRUE"
+#DO_ENSFCST="TRUE"
+#DO_DACYCLE="TRUE"
+#DO_SURFACE_CYCLE="TRUE"
+DO_SPINUP="TRUE"
+DO_SAVE_INPUT="TRUE"
```

Comment:
Do you need to save INPUT/ for the ensemble run? For ensemble cycling runs, this should be set to "FALSE". Add "DO_SAVE_DA_OUTPUT" to "TRUE" if you want to save the DA analysis, or set it "FALSE" if you don't save them in real-time runs.

Reply:
Yes, I am confused with those.

Reply:
Changed